lerna.json

```diff
@@ -1,4 +1,4 @@
 {
-	"packages": [ "packages/*", "client/gutenberg/extensions/presets/jetpack" ],
```

Comment:
👍 We'll sort out `jetpack-blocks` independently. Still a lot of movement there and it's not a traditional npm package.